modules/labs-react/form-field/stories/examples/Integrating.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {FormField} from '@workday/canvas-kit-labs-react/form-field';
+import {VStack} from '@workday/canvas-kit-labs-react/layout';
+import Input from '@material-ui/core/Input';
```

Comment:
We've been trying to remove `@material-ui/core` from our dependencies. Perhaps the idea is that custom inputs could be used via the `as`?